### PR TITLE
Assign Copyright based on Usage Rights

### PR DESCRIPTION
**Pull request by adrielulanovsky**
## What does this change?
When an image has no copyright and a StaffPhotographer usage right is selected, the copyright will be automatically filled to be the publication of the staff photographer.

Discussion of this change: https://github.com/guardian/grid/discussions/3474

## How can success be measured?
When an image has no copyright and a StaffPhotographer usage right is selected, the copyright should be automatically filled to be the publication of the staff photographer.
When an image has a previous copyright, it should remain unchanged when selecting a StaffPhotographer usage right.

## Who should look at this?
@guardian/digital-cms

## Tested? Documented?
- [x] locally by committer
- [ ] locally by Guardian reviewer
- [ ] on the Guardian's TEST environment
- [ ] relevant documentation added or amended (if needed)

### CODE REVIEW

**Comment by paperboyo** [comment]
For visibility, we found some issues with it, visible in this GIF:
2Untitled

1. `PA Wire` is being replaced by `The Guardian` (it shouldn’t) and subsequently
2. `The Guardian` is not being replaced by `The Observer` (would be nice for all publications to be replaceable)

1 is a bug. 2 would be a nice addition not affecting BBC (only one `publication`?), but helpful for the Guardian.

**Comment by adrielulanovsky** [comment]
@paperboyo Could you try again? Both 1 and 2 should be fixed now.

**Comment by paperboyo** [comment]
Hi, @adrielulanovsky. Tested this now (sorry for the delay) and the point 2 works nice. But pt. 1 is still a problem: existing non-publication Copyright *changes* upon application of Staff Photographer both on the upload page and in Viewer/Browser (see the beginning of teh above GIF).

**Comment by adrielulanovsky** [comment]
@paperboyo Do you mind giving this another try? It should (:crossed_fingers: ) be fixed now.

**Review by andrew-nowak — APPROVED** [review]

**Comment by andrew-nowak** [comment]
implementation LGTM, we're waiting on some feedback from users before merging

**Pull request merged**
